scripts/jobs/cron_tasks.php

```diff
@@ -51,6 +52,14 @@
 	 */
 	if ($row['type'] == '1') {
 
+		// get class for phpfpm-restart
+		$phpfpm_restart = new phpfpm_restart($cronlog);
+
+		// if phpfpm is enabled, capture current php-configs
+		if (Settings::Get('phpfpm.enabled') == 1) {
+			$phpfpm_restart->captureCurrentConfigs();
```

Comment:
This is required if e.g. multiple configs use the same restart-script and to restart only servers which are used and leave unused ones untouched. It seems weird because it was planned otherwise, I will change some things